### DISCUSSION TITLE
Every time I star a project, it gets automatically unstarred immediately. I must star it again.

### DISCUSSION THREAD

**Discussion by lfjmgs** [community/community#61305] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.122.0 (104639223)
Device: iPhone 13 mini (iOS 16.5.1)
</details>

**Comment by esahmed21** [comment]
Hi @lfjmgs 👋

Thank you for reaching out! I weren't able to reproduce this so Would you mind share specific scenario where this happens? And if it happens with public repository, would be great if you share it so that i can help you.

Thanks!

**Reply by lfjmgs** [reply]
https://github.com/community/community/assets/21254187/8815d737-0474-4617-9165-a271ced1e968
1. navigated to the repository page, then starred the repository, but it was automatically unstarred immediately.
2. Went back to the previous page, then navigated to the repository page again, but the repository is starred. Unstarred the repository, but it was automatically starred immediately.
3. Went back to the previous page again, then navigated to the repository page again, but it was unstarred automatically.
